Src/Plugins/Security/SEC101_006.GitHubPatValidator.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Linq;
 
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.HelpersUtiliesAndExtensions;
```

Comment:
>HelpersUtiliesAndExtensions [](start = 67, length = 27)

check if we are using...

Reply:
Removed.

---
In reply to: [575575569](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575575569) [](ancestors = 575575569)